content/guide/plugins-authoring.md

```diff
@@ -298,7 +298,65 @@ def start(**kwargs):
 
 # Testing Your Plugin
 
-In most cases, the recommendation is to test your plugin's logic using local workflows and only then, run them as part of a Cloudify [deployment]({{page.terminology_link}}#deployment). The [Plugin Template]({{page.template_link}}/blob/3.2/plugin/tests/test_plugin.py) has an example of doing just that.
+In most cases, the recommendation is to test your plugin's logic using local workflows and only then, run them as part of a Cloudify [deployment]({{page.terminology_link}}#deployment). We have supplied you with a nice and tidy
+decorator to do just that. It is provided by the `cloudify-plugins-common.test_utils`, and it's very intuitive to use, But just in case let's look at an example:
```

Comment:
the 'cloudify-plugins-common' thing looks like a python path, but it's got hyphens and all.. should either be a non-code "cloudify-plugins-common's test_utils package", or the python path of the package